### ISSUE TITLE
NVDA not narrating "radio button" selected access text after selecting the radio button

### ISSUE THREAD

**Issue by subbart** [nvaccess/nvda#9628]
### Actual behavior:
1) Open WPF window with radio buttons
2) Navigate the radio button using nvda keys
3) NVDA currently narrates the current status of radio button ( enabled / disabled)
4) Now select or deselect the radio button
5) NVDA not narrating new state
6) Go to any other control on the page and come back , NVDA correctly narrates the new state.

Microsoft narrator is narrating new state immediately after state change , step 5) above.

### Expected behavior:
In step 5) above NVDA should announce new state

Microsoft narrator is narrating new state immediately after state change , step 5) above.

#### NVDA version:
2019.1.1
#### Windows version:
Windows 10 1809 Update
#### Name and version of other software in use when reproducing the issue:
System center virtual machine manager 2019
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

**Comment by Adriani90** [comment]
@subbart are you still having this issue with NVDA 2020.1?

**Comment by Adriani90** [comment]
I am closing this issue as abandoned due to missing updates from the initial author. Please comment here if you are still having this issue and we can reopen.
